src/hook.c

```diff
@@ -433,15 +448,26 @@ static VOID EnterSpinLock(VOID)
     SIZE_T spinCount = 0;
 
     // Wait until the flag is FALSE.
+#ifndef _MSC_VER
+    while (InterlockedCompareExchange(&g_isLocked, TRUE, FALSE) != FALSE)
+    {
+#else
     while (_InterlockedCompareExchange(&g_isLocked, TRUE, FALSE) != FALSE)
     {
         _ReadWriteBarrier();
-
+#endif
         // Prevent the loop from being too busy.
+#ifndef _MSC_VER
+        if (spinCount < 16)
+            usleep(250);
```

Comment:
Why can't you use the same code for GCC?
Seems like `_mm_pause` is supported.
http://stackoverflow.com/a/7372129